samples/matter/light_switch/README.rst

```diff
@@ -21,34 +21,55 @@ The sample supports the following development kits:
    :header: heading
    :rows: nrf52840dk_nrf52840, nrf5340dk_nrf5340_cpuapp, nrf21540dk_nrf52840
 
-For this sample to work, you also need the :ref:`Matter light bulb <matter_light_bulb_sample>` sample programmed to another supported development kit.
+For this sample to work, you also need at least one :ref:`Matter light bulb <matter_light_bulb_sample>` sample programmed to another supported development kit.
+
+To commission device and do all commands needed to prepare all device to work you need also a controller. 
+You can use a `Chip-Tool <https://github.com/project-chip/connectedhomeip/tree/master/docs/guides/chip_tool_guide.md>`_ to perform commissioning and prepare all devices to work.
```

Comment:
The doc still doesn't exist, so I suggest removing it and adding when it's available.